### ISSUE TITLE
Farmer wont do anything

### ISSUE THREAD

**Issue by TerraStellaVltra** [ldtteam/minecolonies#2306]
### Minecolonies version
1.12.2-0.8.7719

version X.X.XXXX or curse X.X.XX

### Expected behavior
farmer to til, plant and harvest things.

- XXX

### Actual behaviour
he just stands there and does nothing.
- YYY

### Steps to reproduce the problem
1. hire the farmer
2. I place the feild
3. place the seed inside the scarecrow
4. I give the farmer a stack of seeds and a stone hoe
5. he walks to the farm chest, then to the field then just stands there

I've tried restarting the game, recalling the towns people, firing and rehiring, everything. 
Theres no water underneath the fields either as I seen that was an issue once.

**Comment by Sandriell** [comment]
@TerraStellaVltra Since you closed this, I assume you figured it out. You should post how you fixed it for anyone else that may have the problem and comes here searching for an answer.

I refer you to this comic for why: https://xkcd.com/979/ :D

**Comment by TerraStellaVltra** [comment]
I planned on writing when I figured out what happened. I'm still not sure so I'll give it a go. 

When I started with this problem, I had a slightly older version of 1.12.2. I downloaded a newer version. But he still appeared to not be working. That's when I wrote this issue to github. He wouldn't do anything for a while, a long while. I gave him an iron hoe, though I'm not sure that helped. But eventually he started working, slowly, but he's working. I imagine he'll become more efficient with time as I upgrade the farm. We'll see. He also complained of not sleeping though he just started his job. So I made sure to also not sleep and do something else to let my NPC's sleep. Again, I'm not sure if that helped. 

That's the best I've got really. I'm not sure which thing, if not everything, I've tried that fixed the issue, though. That's why I didn't write anything. In fact, I was waiting to see if he stopped again after a few game days. He hasn't, thankfully so.
